testsuite/thermo_integration/test_thermo_integration.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import sys
+import time
+import espressopp
+import mpi4py.MPI as MPI
+
+import unittest
+
+class TestThermoIntegration(unittest.TestCase):
+    def setUp(self):
+
```

Comment:
@fedepad I was thinking the same thing as you, but actually this file test_thermo_integration.py is copied directly from @MrTheodor 's test_minimize_energy.py, which also has all the same empty lines. I think he might be suggesting to add more empty lines before and after the class definition?!